doc/workshops/tutorial/content/examples/tutorial3.md

```diff
@@ -0,0 +1,74 @@
+# Tutorial 3 - Electric Field Driven Ion Loss
+
+!---
+
+# style=margin-top:2em;
+
+# Problem Statement
+
+Consider a single species plasma, an ionic Argon plasma, in a one-dimensional system on the domain $x\in[0, l]$. In this system there is an initial uniform distribution of singly charged Argon ions. This is the same problem as described on page 26-27 of [!cite](lieberman1994principles). This plasma is subject to the electric field generated by its own charge density. Additionally, the walls of this system destroy ions.
+
+# style=margin-top:3.75em;
+
+!bibtex bibliography
+
+!---
+
+# Sytem of Equations
+
+# style=margin-top:2em;
+
+\begin{equation}
+  \pf{n_i}{t} +
+  \nabla \cdot
+  \left[
+  \left(
+    \mu_i
+    \nabla
+    \phi
+  \right)
+  n_i
+  -
+  D_i
+  \nabla
+  n_i
+  \right]
+  =
+  0
+\end{equation}
+
+
+\begin{equation}
+  - \nabla^2
+  \phi
+  =
+  \frac{e n_i}{\varepsilon_0}
+\end{equation}
+
+\begin{equation}
+  n_i(x, 0) = n_{i0}
+\end{equation}
+
+\begin{equation}
+  n_i(0, t) =
+  n_i(l, t) =
+  0
+\end{equation}
+
+!---
+
+# style=margin-top:2em;
+
+### What would happen ot the rate of the potential decay as pressure changes?
+
+!listing tutorial/tutorial03-PotentialWithIonLoss/ion-loss-for-IonOnlyPlasma.i block=Materials/GasBasics
+
+!alert note
```

Comment:
```suggestion
!style halign=left
!alert note
```

I would try using this trick to add a little bit of space between the listing and the alert.